addons/web_editor/static/lib/odoo-editor/src/powerbox/Powerbox.js

```diff
@@ -352,7 +352,7 @@ export class Powerbox {
     }
 
     _resetPosition() {
-        const position = getRangePosition(this.el, this.options.document, this.options);
+        const position = getRangePosition(this.el, this.options.document, { parentContextRect: this.options.getContextFromParentRect() });
```

Comment:
Was this for sure the only option we were interested in? This seems like an unrelated and possibly breaking change.

Reply:
Till now, Yes! Actually it is a fix for a bug. See this [commit](https://github.com/odoo/odoo/pull/104960/commits/8795d335643a9ed7bbac7df046e1abef1878596a). When the function was originally created we needed four margin values as options which we have not used yet (Might be because we didn't find need for it). Right now the same bug can be reproduced in case of table picker, [See](https://www.awesomescreenshot.com/image/46172078?key=fbea4539921d7a8e89729431053378ed). I have tried to cover it by doing the same thing. What do you think would be the best alternate action here?

Reply:
What were we passing in `this.options`? Obviously `getContextFromParentRect` was already in it, so is this meant to remove the margin values? I don't understand what this change does.